djangosaml2idp/views.py

```diff
@@ -142,11 +153,13 @@ def get(self, request, *args, **kwargs):
         AUTHN_BROKER = AuthnBroker()
         AUTHN_BROKER.add(authn_context_class_ref(req_authn_context), "")
 
+        user_id = processor.get_user_id(request.user)
+
         # Construct SamlResponse message
         try:
             authn_resp = self.IDP.create_authn_response(
-                identity=identity, userid=request.user.username,
-                name_id=NameID(format=resp_args['name_id_policy'].format, sp_name_qualifier=resp_args['sp_entity_id'], text=request.user.username),
+                identity=identity, userid=user_id,
+                name_id=NameID(format=resp_args['name_id_policy'].format, sp_name_qualifier=resp_args['sp_entity_id'], text=user_id),
```

Comment:
As an fyi: this was changed from `sp_name_qualifier=resp_args['destination']` to `sp_name_qualifier=resp_args['sp_entity_id']` according to the issue filed here https://github.com/OTA-Insight/djangosaml2idp/issues/18

Reply:
OK, makes sense - I'll test this in our UAT env tomorrow.